packages/terra-switch/tests/wdio/switch-spec.js

```diff
@@ -0,0 +1,45 @@
+Terra.describeViewports('Switch', ['tiny', 'medium', 'large'], () => {
+  describe('DefaultSwitch', () => {
+    before(() => browser.url('/#/raw/tests/terra-switch/switch/default-switch'));
+
+    Terra.it.validatesElement();
+
+    describe('Focus', () => {
+      before(() => {
+        browser.keys('Tab');
+      });
+
+      Terra.it.validatesElement();
+    });
+
+    describe('Checked', () => {
+      before(() => {
+        browser.waitForVisible('#defaultSwitch');
+        browser.click('[for="defaultSwitch"]');
```

Comment:
Should we add a data-attribute on the slide that can be used in tests by both us and our consumers?

Reply:
but we already have required props for both labelId and switchId(buttonId) do we still need data-attribute..?

Reply:
oh no 👍 I didn't realize that this mapped to the id since I looked at this first. So technically this would be equivalent right?
```
browser.click('#defaultSwitch');
```

Reply:
yes it is, here i am just trying ensure switch gets checked on clicking label.